File: 백준/Bronze/9610. 사분면/사분면.py
import sys

input = sys.stdin.readline

n = int(input().strip())
q1, q2, q3, q4, AXIS = 0, 0, 0, 0, 0

for _ in range(n):
    x, y = map(int, input().strip().split())
    if x * y == 0:
        AXIS += 1
    elif x > 0 and y > 0:
        q1 += 1
    elif x < 0 and y > 0:
        q2 += 1
    elif x < 0 and y < 0:
        q3 += 1
    elif x > 0 and y < 0:
        q4 += 1

sys.stdout.write(f"Q1: {q1}\nQ2: {q2}\nQ3: {q3}\nQ4: {q4}\nAXIS: {AXIS}\n")

File: 백준/Bronze/9610. 사분면/사분면.java
import java.io.*;

public class Main{
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
		
        int n = Integer.parseInt(br.readLine().trim());
        int q1 = 0, q2 = 0, q3 = 0, q4 = 0, AXIS = 0;

        for (int i = 0; i < n; i++){
            String[] input = br.readLine().trim().split(" ");
            int x = Integer.parseInt(input[0]);
            int y = Integer.parseInt(input[1]);
            if (x * y == 0) {
                AXIS++;
            }else if(x > 0 && y > 0) {
                q1++;
            }else if(x < 0 && y > 0) {
                q2++;
            }else if(x < 0 && y < 0) {
                q3++;
            }else if(x > 0 && y < 0) {
                q4++;
            }
        }

		bw.write("Q1: " + String.valueOf(q1) + '\n');
		bw.write("Q2: " + String.valueOf(q2) + '\n');
		bw.write("Q3: " + String.valueOf(q3) + '\n');
		bw.write("Q4: " + String.valueOf(q4) + '\n');
		bw.write("AXIS: " + String.valueOf(AXIS) + '\n');
		bw.flush();
		bw.close();
		br.close();
	}
}